Fix issue with autologin with LDAP and user email

piti-diablotin opened the pull request:
This is a fix to add an option in the admin panel.
Now the checkbox for automatically login users is a 3 radio buttons.
* no automatic login
* automatic login with NC credentials like user@domain
* automatic login with NC user email which can be whatever. In this case, still the NC password is used to try the login. This option is used in my case for LDAP user with an email which is on the same domain as NC but not like user@domain.
See issues #11 and #88 for more details.

The three options have been tested locally but not the "user defined email and password" in the personnal settings.

I think the way it is implemented will not break backward compatibility.

Any review/comments are welcome.

PS:the pl translation has not been updated.
PPS: I can add the Fr translation if desired.

Commit:
Fix issue with autologin with LDAP and user email differente than NC user@domain. See issue #11 and #88

benibr commented on rainloop/app.php:
```diff
@@ -36,19 +36,28 @@
 		else
 		{
 			$sEmail = \OC::$server->getConfig()->getUserValue($sUser, 'rainloop', 'rainloop-email', '');
+      $sPasswordSalt = $sEmail
```
Here a semicolon is missing at the end of the line. That crashes the app right on start.

benibr commented:
Hi, thanks implementing! I am really looking forward this feature. I tried this on my running NC16
and had to fix a missing semicolon, see https://github.com/pierre-alain-b/rainloop-nextcloud/pull/111/files#r338681990
After that, logging in with email set in nextcloud account works like charm :-)

What else is needed to merge this?

Commit:
Fix missing coma.
Beautification for tabs alignment
Fix autologin with autologin with email which did not work ...

piti-diablotin commented:
Thanks for reporting !
Please try new version since last one did not work (except if you were already logged in with the old autologin).
Check it out in a private navigator tab !
Cheers

benibr commented on rainloop/app.php:
```diff
@@ -1,4 +1,4 @@
-<?php
+appinfo/app.php<?php
```
This is most probably not meant to be there in der PR?

benibr commented:
except for the first line, it works!

Commit:
Fix typo again ..

piti-diablotin commented:
Sorry for this typo !
It is now fixed.

benibr commented:
@pierre-alain-b any objection or can this be merged?

pierre-alain-b commented:
I will review and then merge

benibr commented:
Hi everyone and happy hollidays! I still use this branch and like to go back to master. Anything needed to merge this?
